test/core/consensus/babe/block_executor_test.cpp

```diff
@@ -240,6 +235,9 @@ TEST_F(BlockExecutorTest, JustificationFollowDigests) {
   EXPECT_CALL(*consistency_keeper_, rollback(block_info))
       .WillRepeatedly(Return());
 
-  EXPECT_OUTCOME_TRUE_1(block_executor_->applyBlock(
-      Block{block_data.header.value(), block_data.body.value()}, justification))
+block_executor_->applyBlock(
+      Block{block_data.header.value(), block_data.body.value()}, justification, [](auto &&result) {
+  EXPECT_OUTCOME_TRUE_1(result)      ;
```

Comment:
looks non-formatted. and likely it will not work - this macro works only in test-body